server/config/index.js

```diff
@@ -17,6 +18,16 @@ if (process.env.GRAPHIQL_QUERY_FILE) {
   }
 }
 
+if (process.env.GRAPHIQL_VARIABLES_FILE) {
+  try {
+    graphiqlVariableFileContent = fs.readFileSync(process.env.GRAPHIQL_VARIABLES_FILE, 'utf-8')
+    graphiqlVariableFileContent = JSON.parse(graphiqlVariableFileContent)
```

Comment:
The encoding type may be used in others place and could be changed, in this way, what do think about centralized it in a var ?

Reply:
For now, we can create a constant in the same file as I don't think we don't have any other place that we use an encoding.